### PR TITLE
Feature/application notes

### PR DESCRIPTION
**Pull request by rdperry031**
## Description

A notes text input field has been added to the pending applications modal in order to add notes to any particular application. The field is hidden when the modal is rendered and displayed only after the "Edit Notes" button is clicked. The field is a textarea rather than a simple text input in order to display the notes in their entirety and therefore making adding/editing them easier. We also snuck in a few minor tweaks to the PendingApplications component.

## Video

https://www.loom.com/share/8befbff4a9e64bf08af9a49fa1c6dc65

## Type of Change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Review by mktran0417 — APPROVED** [review]
I think the only major thing that needs to be changed is the video is relatively hard to follow along to in terms of what's happening. The text is rather aliased.

**Review by JohnMBarnett — APPROVED** [review]
Looks great! Very good implementation.

**Review by djho57 — APPROVED** [review]
Changes look good. Video was a bit hard to follow.

File: src/components/pages/PendingApplications/PendingApplication.css [hunk 1]
```diff
@@ -14,7 +14,11 @@
   border: none;
   color: white;
   padding: 0.5rem;
+<<<<<<< HEAD
```
Comment (paulstgermain) [inline src/components/pages/PendingApplications/PendingApplication.css:17]: It looks like you somehow included an unresolved merge conflict message in your PR? Let's make sure to resolve this and then push those changes up so we don't have any production crashes!

**Review by paulstgermain — CHANGES_REQUESTED** [review]
This PR is looking good so far - cleanly implemented, and a much-needed change in order to sync our forms up with our BE. The only thing that needs to be changed here is the included merge conflict message - resolve that and update this branch and this should be good to go!

**Review by paulstgermain — APPROVED** [review]
